### PR TITLE
Compile Fix

### PR DESCRIPTION
**Pull request by DustinGrau-PSC**
# Description

Adds additional error handling for creating the callback class for 11.3+ installations.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [-] New feature (non-breaking change which adds functionality)
- [-] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [-] This change requires a documentation update

# Checklist:

- [X] My branch is started from the latest commit in main
- [-] My commit history is clean
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [-] I have made corresponding changes to the documentation
- [-] My changes generate no new warnings
- [-] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/progress/pct/compile.p [hunk 1]
```diff
@@ -212,15 +212,33 @@ END PROCEDURE.
 PROCEDURE initModule:
   ASSIGN lIgnoredIncludes = (LENGTH(cignoredIncludes) > 0).
 
-  IF (callbackClass > "") AND NOT bAboveEq113 THEN
+  IF (callbackClass > "") EQ TRUE AND NOT bAboveEq113 THEN
```
Comment (gquerret) [inline src/progress/pct/compile.p]: @DustinGrau-PSC What's that ?

Comment (DustinGrau-PSC) [reply src/progress/pct/compile.p]: That should handle the case where callbackClass may somehow be the unknown value ? as well as an empty string. It ensures that the value is truly a string.

Comment (gquerret) [reply src/progress/pct/compile.p]: I've tried with multiple values (`?`, empty string, ...) and the `eq true` is not necessary

Comment (DustinGrau-PSC) [reply src/progress/pct/compile.p]: Removed. It was more of a suggestion/habit when I use the > "" syntax for string checks.

**Comment by DustinGrau-PSC** [comment]
Merci! Will there be a new release version created, or does that only occur when there are significant updates?

**Comment by gquerret** [comment]
v227 was released last week, so no, there won't be a new release soon. You can always download builds here: https://ci.rssw.eu/job/PCT/job/main/

**Pull request merged**